### DISCUSSION TITLE
dataspace connector deployment using kubernetes manifests.

### DISCUSSION THREAD

**Discussion by sohailkooltech** [International-Data-Spaces-Association/DataspaceConnector#258] [Q&A]
I am trying to deploy dataspace kubernetes Full setup. However, you didnt provide kubernetes manifests for deployement. 
So I have created my own and trying to deploy. 
I have make one secrets file and put all the base64 encoded values mentioned in env files for dataspaceconnector, ui and postgress and use that secrets in all the deployements. 
However I am facing one error mentioned in the screenshot CrashBackLoopOff but I didnt get the actual reason of this. Please take a look and see if you can suggest me something.
![image](https://user-images.githubusercontent.com/103914210/169276905-c1972571-0507-45e3-ac86-f463a087c348.png)

**Comment by ghost** [comment]
Hello, after first look something seems to be wrong with your PostgreSQL setup, at least that looks like the cause of the error message, see last line.

Here's a short docu how to setup the DSC using Helm:
https://github.com/International-Data-Spaces-Association/IDS-Deployment-Examples/tree/main/dataspace-connector/slim/k8s

And here are the charts:
https://github.com/International-Data-Spaces-Association/DataspaceConnector/tree/main/charts/dataspace-connector

**Reply by sohailkooltech** [reply]
This links you have mentioned just provide the slim setup but I have to use full setup.

**Reply by sohailkooltech** [reply]
seems my postgres working properly. 
![image](https://user-images.githubusercontent.com/103914210/169291402-74733060-aee7-4c25-b045-cc8805c14eeb.png)

**Reply by sohailkooltech** [reply]
![image](https://user-images.githubusercontent.com/103914210/169291613-e34c9cf6-f398-4293-9c18-0830a6ce50a7.png)

**Reply by ghost** [reply]
I will forward this internally but I have to ask for a little patience, insofar as our help should still be required later.

**Reply by sohailkooltech** [reply]
Thanks, let me know if you need the manifests I have prepared I will share with you.

**Reply by sohailkooltech** [reply]
@tmberthold  Any update?
